src/index.html

```diff
@@ -175,7 +175,11 @@ <h1 class="text-xl/7 font-heading font-[375]" data-i18n="site_title">
           <div id="history" class="dark:text-white" data-testid="history"></div>
         </div>
       </section>
-      <section class="bg-blue-900">
+      <section
+        class="bg-blue-900"
+        aria-label="Go to GC Notify"
+        data-i18n="back_button"
```

Comment:
Probably a copy and paste issue: this is causing the i18n library to replace the contents of this element with the localized string

Reply:
ugh, I know what happened, I was hoping it would target the aria-label, but I definitely doesn't!